### ISSUE TITLE
[BUG] - Linux Appimage and pacman package crash on startup. "GPU process isn't usable. Goodbye."

### ISSUE THREAD

**Issue by Cknight70** [ebkr/r2modmanPlus#1395]
**Describe the bug**
When starting r2modmanPlus with both the appimage version and the pacman package, the program will start and show a blank r2modmanPlus window for a few seconds, then close.

**To Reproduce**
1. Install the pacman package, or simply run the appimage provided in releases or on the thunderstore page
2. Run program

**Additional context**
I have a Nvidia graphics card, I'm using the proprietary driver on Xorg, I am Running Arch linux.

I ran the app image in bash and got the following log
```
$   /ebkr-r2modman-3.1.49/r2modman-3.1.49.AppImage 
xdg-mime: application argument missing
Try 'xdg-mime --help' for more information.
starting server on  /tmp/app.r2mm 
starting TLS server false
starting server as Unix || Windows Socket
[7373:0707/182304.968192:FATAL:gpu_data_manager_impl_private.cc(445)] GPU process isn't usable. Goodbye.
Trace/breakpoint trap (core dumped)
```

**Comment by Cknight70** [comment]
Hi, I've seemed to have fixed the issue myself by appending `--no-sandbox` at the end of the run command for this. 
so for example I can type `r2modman --no-sandbox` in bash, or add the argument to the desktop startup file like this screenshot

![image](https://github.com/ebkr/r2modmanPlus/assets/28763729/044b44e6-1506-4c93-9fe8-f2f95256e674)

**Comment by Jaskowicz1** [comment]
Same issue with the deb installation, just seems like it's every single package that is broken.

**Comment by TupalupaGit** [comment]
Your solution worked for me, thanks!

**Comment by al12gamer** [comment]
Are the people here experiencing this using NVIDIA cards? Honestly wondering if this is a recurring theme as it was in #1383

**Comment by Jaskowicz1** [comment]
@al12gamer yeah I'm on Nvidia getting the issue 😅 the no-sandbox flag does fix this though.

**Comment by al12gamer** [comment]
My friend experiencing the issue has NVIDIA as well. I haven't experienced this on the Intel test bench nor AMD gaming desktop so I'm wondering if there's something to that.

**Comment by swapmasterx** [comment]
I'm getting this issue with my deb install. This did fix it

**Comment by sageworksstudio** [comment]
Same .deb install issue (with Nvidia graphics), @Cknight70 's suggestion to use the `--no-sandbox` option worked to get it started.

**Comment by candroid-man** [comment]
How do I add the `--no-sandbox` flag on Fedora on GNOME? I'd like it to run that flag any time I launch the RPM package

**Comment by Cknight70** [comment]
@candroid-man 

Find your .desktop file in

/usr/share/applications/ or /usr/local/share/applications/ for applications installed system-wide, or ~/.local/share/applications/

Find the "exec" line, and add the --no-sandbox  parameter on the same line. So it may look like `Exec=r2modman --no-sandbox`

**Comment by OliverKreitmeyer** [comment]
Same issue on Fedora 40 Workstation on both rpm and appimage versions. Works when launched with `--no-sandbox`. Also on Nvidia GPU with proprietary drivers.
